metrics/src/main/java/com/expedia/adaptivealerting/metrics/functions/source/MetricFunctionsReader.java

```diff
@@ -30,8 +32,9 @@
 
     public static List<MetricFunctionsSpec> readFromInputFile(String InputFilename) {
         List<MetricFunctionsSpec> metricFunctionSpecList = new ArrayList<>();
-        InputStream inputStream = MetricFunctionsReader.class.getResourceAsStream(InputFilename);
+        String inputFile = new File(InputFilename).toString();
         try {
+            InputStream inputStream = new FileInputStream(inputFile);
```

Comment:
can't we use InputFilename parameter directly in new FileInputStream(). e.g new FileInputStream(InputFilename);

Reply:
Fixed.